### ISSUE TITLE
Request for project inclusion: skdag

### ISSUE THREAD

**Issue by big-o** [scikit-learn-contrib/scikit-learn-contrib#56]
Request for project inclusion in scikit-learn-contrib

* Project name: skdag
* Project description: Directed acyclic graphs as metaestimators. `skdag` defines a new metaestimator, `DAG`, which works in a similar way to a pipeline except it supports much more complex workflows including advanced preprocessing, model stacking, benchmarking etc. Unlike many other requests to scikit-learn-contrib, this project does not implement any new research. Instead, it provides a convenient implementation of industry-standard practices that is currently missing from scikit-learn and requires a non-trivial amount of work from the user to implement.
* Authors: https://github.com/big-o
* Current repository: https://github.com/big-o/skdag
* Requirements:
 * [x] scikit-learn compatible (check_estimator passed)
 * [x] Documentation (guide, API reference, example gallery) - https://skdag.readthedocs.io/
 * [x] Unit tests (coverage: 91%)
 * [x] Python3 compatible
 * [x] PEP8 compliant
 * [x] Continuous integration

**Comment by big-o** [comment]
Hi, just wanted to check if you have any thoughts on this?

**Comment by adrinjalali** [comment]
This looks pretty descent to me! If there are no objections by others, I'm happy to have it moved under contrib.

**Comment by big-o** [comment]
> This looks pretty descent to me! If there are no objections by others, I'm happy to have it moved under contrib.

Thanks! Just let me know if/when the community is happy and I'll do what is needed at my end for the transfer.

**Comment by adrinjalali** [comment]
Since there are no objections, if you add me as an owner to the repo, I can move it to the scikit-learn-contrib org.

**Comment by big-o** [comment]
> Since there are no objections, if you add me as an owner to the repo, I can move it to the scikit-learn-contrib org.

I've sent you an invite to become a collaborator, hopefully that will give you the access you need? I can't find any options to label someone as an 'owner'.

**Comment by adrinjalali** [comment]
I accepted it. I think you need to make me an "owner" for me to be able to move the repo :)

**Comment by big-o** [comment]
> I accepted it. I think you need to make me an "owner" for me to be able to move the repo :) 

Can you advise on how to do this? As far as I can tell it is not possible to add owners to a GitHub repo, only collaborators:

https://stackoverflow.com/questions/8015579/add-another-user-to-project-owners-in-github

I would give you more access if I could, but I can't find any other options in my project settings to give you any more permissions than I already have.

Can you just create a new clone of the repo in your scikit-learn-contrib project? Or perhaps create a new empty repo in your project and then give me permission to push to it, and I can copy the code and commit history across myself?

**Comment by adrinjalali** [comment]
It's better to transfer the project since it'd transfer the issue/PR history as well.

I think you should be able to make me an "admin" if you go here: https://github.com/big-o/skdag/settings/access

**Comment by big-o** [comment]
I'm afraid I have no such option to make anyone an admin. I've attached a screenshot of everything I have.

![Screenshot_20220830-114601.png](https://user-images.githubusercontent.com/1134151/187417778-5ed37545-5f44-4dff-b3a0-d5185a990ec5.png)

I wonder if this is because my project is an individual one and not part of an organization?

I do however have the ability to move the repo across to a new project myself if I have the correct permissions on the destination project:

![Screenshot_20220830-114910.png](https://user-images.githubusercontent.com/1134151/187418156-4502069a-0c8e-45d4-9b03-8b102c9fad66.png)

**Comment by big-o** [comment]
While I don't have permission to transfer the project to sklearn-contrib, I DO have permission to transfer it to you. I have submitted a request to you to do this and if you approve you'll have ownership. Then hopefully you can move it to the correct project yourself.

**Comment by adrinjalali** [comment]
Done, and you're an admin there now: https://github.com/scikit-learn-contrib/skdag/settings/access

**Comment by big-o** [comment]
Many thanks! Is there anything else for me to do now, beyond maintaining the project as before?

**Comment by adrinjalali** [comment]
Nope, thanks for contributing to the community. Feel free to add other maintainers to the project as you deem appropriate. All sklearn core devs have write access as well, but we usually don't interfere with the project.

Also happy to help if you have questions regarding building a community and all.

Happy hacking ❤️

**Comment by big-o** [comment]
@adrinjalali thank you for your support! If you have any pointers on how to build a wider community I would love to hear them. The main reason for me requesting to join the project is exactly this. As an individual without a large following doing this as a side-project, it's difficult to gain the wider awareness needed to build up the user base needed to develop the project further. Hopefully being part of scikit-learn-contrib will help with this :)

**Comment by adrinjalali** [comment]
I've got some resources here, they might help: https://github.com/adrinjalali/talks/blob/master/2022/EuroPython-opensource-community/Product%2C%20Open%20Source%2C%20Community.pdf
